apps/api-videos/src/app/modules/algolia/algolia.consumer.ts

```diff
@@ -0,0 +1,15 @@
+import { Processor, WorkerHost } from '@nestjs/bullmq'
+import { Job } from 'bullmq'
+
+import { AlgoliaService } from './algolia.service'
+
+@Processor('api-videos-aloglia')
```

Comment:
```suggestion
@Processor('api-videos-algolia')
```

will need to update everywhere else is says aloglia